pkg/providers/ovirt/mapper/mapper.go

```diff
@@ -610,6 +619,19 @@ func (o *OvirtMapper) mapGraphicalConsoles() *kubevirtv1.Devices {
 	return devices
 }
 
+func (o *OvirtMapper) prepareTabletInputDevice(os string) kubevirtv1.Input {
+	tablet := kubevirtv1.Input{
+		Type: "tablet",
+		Name: "tablet",
+	}
+	if strings.Contains(os, "win") {
```

Comment:
There could be case sensitivity issue here. Wouldn't be better to use https://golang.org/pkg/strings/#EqualFold?

Reply:
:+1: Thank you!